Fix to show "Choose file" button when the format is byte

lsarni opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->
Now the button "Choose type" will be shown when the format is `byte` like in:
```
   requestBody:
       content:
         multipart/form-data:
           schema:
             type: object
             properties:
               base64Image:
                 type: string
                 format: byte
```


### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Use the magic "Fixes #1234" format, so the issues are -->
<!--- automatically closed when this PR is merged. -->
Fixes #6361 


### How Has This Been Tested?
<!--- Please describe in detail how you manually tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
Tested locally using this yaml:
```
openapi: "3.0.0"
info:
  version: 1.0.0
  title: Swagger Petstore
  license:
    name: MIT
servers:
  - url: http://petstore.swagger.io/v1
paths:
  /pets/{petId}/image:
    post:
      summary: Add a new pet image
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                base64Image:
                  type: string
                  format: byte
      responses:
        '201':
          description: Created
```


### Screenshots (if appropriate):
**Before**
![image](https://user-images.githubusercontent.com/15196342/95072282-3f1f9080-070b-11eb-95b1-b70043eae9f1.png)

**After**
![image](https://user-images.githubusercontent.com/15196342/95072159-113a4c00-070b-11eb-8e12-e1146a2d34e7.png)



## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

### My PR contains... 
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

**I think I might need to change the documentation here https://swagger.io/docs/specification/describing-request-body/multipart-requests/ on the table under "Specifying Content-Type" to include byte but I'm not sure. Should I?**

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [x] My changes can and should be tested by unit and/or integration tests.
- [x] If yes to above: I have added tests to cover my changes.
- [x] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

tim-lai commented:
@lsarni Thanks for the PR! However, it needs to include a Cypress test that uses the sample yaml and tests the new behavior.

lsarni commented:
@tim-lai thanks for letting me know about the missing test, I hadn't looked in detail at what kind of testing was being done in the project. I just added a new test and rebased.

tim-lai commented:
@lsarni Upon further review, although this fix is a valid solution to the original issue, the original issue also isn't something currently supported by the OpenAPI specification. I updated and commented on the original issue. Thanks for your time and effort on this, and sorry for any inconvenience.
